skyportal/tests/frontend/test_reminders.py

```diff
@@ -92,7 +92,15 @@ def post_and_verify_reminder_frontend(driver, reminder_text):
     )
     reminder_text_2 = str(uuid.uuid4())
     driver.wait_for_xpath('//*[@id="root_text"]').send_keys(reminder_text_2)
-    next_reminder = (datetime.now() + timedelta(days=1)).strftime("%m/%d/%YT%I:%M %p")
+    first_of_the_month = int((datetime.now() + timedelta(days=1)).strftime("%d")) == 1
+    if first_of_the_month:
+        next_reminder = (datetime.now() + timedelta(days=14)).strftime(
+            "%m/%d/%YT%I:%M %p"
+        )
+    else:
+        next_reminder = (datetime.now() + timedelta(days=1)).strftime(
+            "%m/%d/%YT%I:%M %p"
+        )
```

Comment:
I don't understand why this is needed.

Reply:
because it defaults to 01 and wont input correctly if you put in a 01 again. It's super weird (and totally artificial as a selenium thing)